components/search/bills/BillSearch.tsx

```diff
@@ -30,6 +31,31 @@ const searchClient = new TypesenseInstantSearchAdapter({
   }
 }).searchClient
 
+const extractLastSegmentOfRefinements = (items: any[]) => {
+  return items.map(item => {
+    console.log(item)
+    if (item.label != "topics.lvl1") return item
```

Comment:
Not a blocker, just curious: Is there a type field on `item` that would identify hierarchical facets so we could make this more generic? (e.g. if `item`'s type is actually something like `CurrentRefinementsConnectorParamsRefinement`, it would have a `type` enum field that has a `hierarchical` option - we'd have to test to see if that applies in our case, but it'd be preferable to not have to call out specific fields if it's a straightforward adjustment).

Reply:
For posterity - we addresses this offline and it seems this info is sadly not readily available in the state we'd like. This is fine as is.